Tools-Override/tests.targets

```diff
@@ -28,13 +28,15 @@
 
   <PropertyGroup>
     <XunitRuntimeConfig>$(ToolsDir)\xunit.console.netcore.runtimeconfig.json</XunitRuntimeConfig>
-    <TestRuntimeEnvVar Condition="'$(OS)' == 'Windows_NT'">%RUNTIME_PATH%\</TestRuntimeEnvVar>
+    <TestRuntimeEnvVar Condition="'$(OS)' == 'Windows_NT' and ('$(BuildingInsideVisualStudio)' != 'true' or '$(RUNTIME_PATH)' != '')">%RUNTIME_PATH%\</TestRuntimeEnvVar>
     <TestRuntimeEnvVar Condition="'$(OS)' != 'Windows_NT'">$RUNTIME_PATH/</TestRuntimeEnvVar>
+    <TestRuntimeEnvVar Condition="'$(TestRuntimeEnvVar)' == ''">$(TestSharedFxDir)\</TestRuntimeEnvVar>
```

Comment:
To be clear: All I really care is that we minimize the chance of introducing new breaks and that the breaks are easily found.